src/client.ts

```diff
@@ -54,6 +56,9 @@ export default class Client implements ClientInterface {
     vscode.window.onDidCloseTerminal((terminal: vscode.Terminal): void => {
       if (terminal === this.terminal) this.terminal = undefined;
     });
+    if (os.platform().indexOf('win') == 0) {
```

Comment:
My understanding is that it will return `win32` (even on 64 bit). Can we use equality here instead?
```typescript
os.platform() === "win32"
```

Reply:
sure